catroid/src/main/java/org/catrobat/catroid/ui/dialogs/OrientationDialog.java

```diff
@@ -135,12 +136,19 @@ protected void handleOkButtonClick() {
 		dismiss();
 	}
 
-	private void loadStageProject() {
-		String templateZipFileName =
-				createLandscapeProject ? templateData.landscapeZipFileName : templateData.portraitZipFileName;
+	private void downloadTemplateFile() {
+		String zipFileUrl = baseUrlForTemplates;
+		zipFileUrl += createLandscapeProject ? templateData.getLandscape() : templateData.getPortrait();
+
+		DownloadTemplateTask downloadTemplateTask = new DownloadTemplateTask(getActivity(), templateData, zipFileUrl);
+		downloadTemplateTask.setOnDownloadTemplateCompleteListener(this);
+		downloadTemplateTask.execute();
+	}
 
-		LoadProjectTask loadProjectTask = new LoadProjectTask(getActivity(), projectName, templateData.templateName,
-				templateZipFileName, false, false);
+	private void loadStageProject() {
```

Comment:
Shouldn't be done in a Dialog. Handle the OK-Button is not path of the dialog...